src/libs/network/NetInfoWrapper.native.test.tsx

```diff
@@ -67,6 +84,60 @@ describe('NetInfoWrapper', () => {
       expect(onInternetConnection).not.toHaveBeenCalled()
       expect(onInternetConnectionLost).toHaveBeenCalledTimes(1)
     })
+
+    it('should log network informations to firebase when wifi is used', async () => {
```

Comment:
Pas besoin de préciser qu'on log sur Firebase (au cas où on passe sur Amplitude :p), should log analytics suffit